darwin/importer/importer.py

```diff
@@ -729,6 +729,58 @@ def _get_overwrite_value(append: bool) -> str:
     return "false" if append else "true"
 
 
+def _parse_empty_masks(
+        annotation: dt.Annotation,
+        rl: dt.Annotation,
+        rl_dense_rle_ids: Optional[Set[str]] = None,
+        rl_dense_rle_ids_frames: Optional[Dict[int, Set[str]]] = None,
+    ):
+    if rl_dense_rle_ids_frames is None and isinstance(annotation, dt.VideoAnnotation):
```

Comment:
The structure of this is sound, and largely speaking I wouldn't change much, but some overall comments:
* It would be good to break the complexity down into smaller functions for the various sub clauses it performs
* Whilst I actually don't mind, PEP8 and `pythonism` would have us prefer `elif` over multiple `if` to ensure multiple blocks don't accidentally get run - where applicable obviously.
* Prefer verbose names over short ones - `rl` just as well be `raster_layer` for readability etc.

Reply:
I have updated the code to only have: if with elif. which simplifies the code. Variable names are more verbose too - so anyone can understand what's happening.